src/components/Autocomplete.vue

```diff
@@ -150,6 +150,11 @@ export default {
         .filter((group) => group.items.length > 0)
     },
   },
+  watch: {
+    query(q) {
+      this.$emit('queryChange', q)
```

Comment:
Use 'update:query' as event name to keep it consistent. Later we might expose query as a prop too